src/graphql.ts

```diff
@@ -18,6 +18,8 @@ const NON_VARIABLE_OPTIONS = [
   "mediaType",
 ];
 
+const GHES_V3_SUFFIX_REGEX = /\/api\/v3$/;
```

Comment:
Should this include an optional trailing slash to be safe?
```suggestion
const GHES_V3_SUFFIX_REGEX = /\/api\/v3\/?$/;
```

Reply:
won't hurt, thanks!